app/config.py

```diff
@@ -14,14 +15,14 @@ class Settings(BaseSettings):
     STATIC_DIR: Path = APP_DIR / 'static'
     TEMPLATE_DIR: Path = APP_DIR / 'templates'
 
-    FASTAPI_PROPERTIES = {
+    FASTAPI_PROPERTIES: Dict[str, Any] = {
```

Comment:
Any reason for using `Dict` here instead of `dict`? Using the capital-letter version was only necessary on Python 3.8 and earlier

Reply:
> Any reason for using `Dict` here instead of `dict`?

No really. Should I change it?

Reply:
Yeah, just change it back to dict and remove the `Dict` import from typing. Other than that, I can get this merged!

Reply:
All right, thanks! :smiley:

Reply:
One more thing, I only added those changes to the `main` branch. Do you want me to add them to the other branches?

Reply:
I was planning to do those anyway, but that'd be great if you have time!

Reply:
Yeah, I'll do it!

Reply:
I have a little issue tho. My fork only has the `main` branch. Do I have to fork each branch by itself?

Reply:
I think i figured it out jajaj. If I find any other problem I'll comment to you :)